lib/tasks/gettext.rake

```diff
@@ -31,18 +31,26 @@ namespace :gettext do
     clean_dir('locale_alaveteli_pro')
   end
 
-  def find(files:, root:)
-    pot_file = File.join(root, "#{text_domain}.pot")
-
-    pot_temp = Tempfile.new(pot_file)
-    pot_temp_path = pot_temp.path
+  def xgettext(pot_file, *files)
+    output = Tempfile.new(pot_file)
+    output_path = output.path
 
     # find new strings and write to temp file
-    GetText::Tools::XGetText.run('--output', pot_temp_path, *files)
+    GetText::Tools::XGetText.run(
+      '--add-comments=TRANSLATORS', '--output', output_path, *files
```

Comment:
Added `add-comments` to retain the existing helpful comment for translators

Reply:
👍  – squash into commit message